fiscalsim_us/tests/microsimulation/data/cps/test_cps.py

```diff
@@ -8,7 +8,24 @@
 
 CPS_YEARS = [CPS_2021, CPS_2022, CPS_2023]
 
+# Temporarily suspending this test because it is failing on GitHub Actions. The
+# failure has to do with the following traceback progression.
+# 1. test_microsim.py::test_microsim_runs_cps calls the Microsimulation() class
+#    from fiscalsim_us/system.py
+# 2. The Microsimulation() class from fiscalsim_us/system.py takes as an input
+#    a CoreMicrosimulation  object defined in the Simulation() class of
+#    policyengine_core/simulations/simulation.py
+# 3. The error is in line 134 of policyengine_core/simulations/simulation.py,
+#    which calls the Dataset method from policyengine_core/data/dataset.py
+# 4. The error is in line 73 of policyengine_core/data/dataset.py  which uses
+#    the download() method in lines 271 to 327. This code tries to download
+#    data with a token that we don't have. Although I don't know why it works

```

Comment:
@rickecon our dataset is public on the GitHub release, which means it's not a token-related issue. I think the issue is GitHub-side; we've had spotty API responses between operating systems.